localstack/services/events/v1/models.py

```diff
@@ -0,0 +1,11 @@
+from typing import Dict
+
+from localstack.services.stores import AccountRegionBundle, BaseStore, LocalAttribute
+
+
+class EventsStore(BaseStore):
```

Comment:
💡 Please check whether persistence still works before merging after moving the model to a new location.

Reply:
This should be done by the persistency test in the companion branch in pro: https://github.com/localstack/localstack-ext/pull/2907
Since all the tests are green including the persistency tests, it should still work after moving.